### ISSUE TITLE
Make buttons on profile pages focusable with the `tab` key

### ISSUE THREAD

**Issue by Chiroyce1** [ScratchAddons/ScratchAddons#4353]
### Is your request related to a problem? Please describe.

In scratch-www pages, the comment `Post` and `Cancel` buttons are all focusable with the tab key, and once they are focused, pressing `enter` clicks the buttons. This is not the case in scratchr2 profile pages. As of now, pressing tab focuses the language drop-down in the footer..

### Describe the solution you'd like

Make these buttons on profile pages focusable with the tab key, and pressing `enter` while the buttons are focused should click them. This is in order to keep things consistent and improve experience for people who use `tab + enter` to post comments.

### Additional context

I figured out that grabbing all the elements and setting the `tabindex` property to `0` on the buttons makes them focusable with the tab key.
```js
let buttons = Array.from(document.getElementsByClassName("button small"));
buttons.forEach(button => button.tabIndex = 0)
```
But they don't seem to be clicked when `enter` is pressed when the button is focused, otherwise I would've made a PR.

**Comment by Norbiros** [comment]
I want to say, that everything works fine when `scratchcr2` and `dark-www` addons are enabled. When only `scratchcr2` is enabled, `TAB` don't work.

IMAO, `dark-www` doesn't add JS, so I think we can fix it in only css...

Also, this JS code doesn't work for me. (I'm using Firefox, Gento Linux)

**Comment by Chiroyce1** [comment]
> Also, this JS code doesn't work for me. (I'm using Firefox, Gento Linux)

Yeah it lets the button be focusable but not clickable.

**Comment by Chiroyce1** [comment]
> I want to say, that everything works fine when `scratchcr2` and `dark-www` addons are enabled

Doesn't seem like that for me

https://user-images.githubusercontent.com/97374054/156697574-07f5461e-2bde-4fae-b555-2ff99f7facb4.mov


Firefox on macOS, SA v1.24.3

**Comment by Norbiros** [comment]
> > Also, this JS code doesn't work for me. (I'm using Firefox, Gento Linux)
> 
> Yeah it lets the button be focusable but not clickable.

It doesn't focus too.

> > I want to say, that everything works fine when `scratchcr2` and `dark-www` addons are enabled
> 
> Doesn't seem like that for me
> Screen.Recording.2022-03-04.at.9.37.01.AM.mov
> 
> Firefox on macOS, SA v1.24.3
IDK, it works fine on mine computer (ups... sometimes. I don't know why, but on some projects it works...)! Also, it focus profiles, not language selection...

Maybe it depends of system? What's your firefox version (mine 95.0.1)?

**Comment by mybearworld** [comment]
It seems to work now, a year later.

**Comment by mxmou** [comment]
> It seems to work now, a year later.

I've always been able to use these buttons with my keyboard on Windows, so this might depend on the browser and OS. In the screen recording it looks like the @Chiroyce1's browser doesn't allow links to be focused (it skips the whole comments section, which contains many links), which would explain this - the buttons are `<a>` elements.

**Comment by mybearworld** [comment]
> > It seems to work now, a year later.
> 
> I've always been able to use these buttons with my keyboard on Windows, so this might depend on the browser and OS. In the screen recording it looks like the @Chiroyce1's browser doesn't allow links to be focused (it skips the whole comments section, which contains many links), which would explain this - the buttons are `<a>` elements.

Oh, okay. This really should be an issue to Scratch then, but given that this is scratchr2, that won't be possible.
